src/App.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Routes, Route } from 'react-router-dom';
+import Navbar from './components/navbar';
+import Books from './components/book';
+import Form from './components/form';
+import Categories from './components/category';
+import './App.css';
+
+function App() {
+  return (
+
+    <>
+      <Navbar />
+      <Routes>
+        <Route
+          path="/"
+          element={(
+            <>
+              <Books />
+              <Form />
+            </>
+)}
+        />
+        <Route path="categories" element={<Categories />} />
+      </Routes>
+
+    </>
+
+  );
+}
```

Comment:
- [ ] Please, make a new "Book" component in a new "Book.js" file, that takes the book data as a prop and renders the title and the author.  The "remove" button should be placed in this file.  

- [ ] Please, kindly make the Form component a child of the "Books" component, import it, and render it from there.

- [ ] The books should be rendered by the "Books" component, using the new "Book" component from the previous step for the book data.